pipelines/pipeline_katello_32.yml

```diff
@@ -32,38 +29,36 @@
     - disable_firewall
     - haveged
     - update_os_packages
-    - foreman_installer
 
 - hosts: pipeline-katello-3.2-centos7
   become: yes
   vars:
     foreman_installer_scenario: katello
-    foreman_installer_options_internal_use_only:
+    foreman_installer_options:
       - "--disable-system-checks"
-      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
-    foreman_installer_additional_packages:
-      - katello
+    installer_scenarios_answers:
+      foreman:
+        admin_password: changeme
+    foreman_installer_pkg: katello
```

Comment:
I never had any need to install more than 1 package, I'll happily change it to take an array if required. I can't see anywhere that this is actually need?